diff --git a/sass.node.mjs b/sass.node.mjs
index f59a3d8987162a3cbc9787c34f67186fb0f09186..b154f46118ccc03989b9a1c11309954c85515539 100644
--- a/sass.node.mjs
+++ b/sass.node.mjs
@@ -40,7 +40,7 @@ export const NodePackageImporter = cjs.NodePackageImporter;
 export const deprecations = cjs.deprecations;
 export const Version = cjs.Version;
 
-let printedDefaultExportDeprecation = false;
+let printedDefaultExportDeprecation = true;
 function defaultExportDeprecation() {
   if (printedDefaultExportDeprecation) return;
   printedDefaultExportDeprecation = true;
