diff --git a/node_modules/@cycjimmy/jsmpeg-player/dist/jsmpeg-player.cjs.js b/node_modules/@cycjimmy/jsmpeg-player/dist/jsmpeg-player.cjs.js
index b527981..6b14518 100644
--- a/node_modules/@cycjimmy/jsmpeg-player/dist/jsmpeg-player.cjs.js
+++ b/node_modules/@cycjimmy/jsmpeg-player/dist/jsmpeg-player.cjs.js
@@ -1160,6 +1160,8 @@ TS.STREAM = {
   DIRECTORY: 0xff
 };
 
+var _jsmpeg_window_reloading = false;
+
 /* eslint class-methods-use-this: ["error", { "exceptMethods": ["destroy"] }] */
 var BaseDecoder = /*#__PURE__*/function () {
   function BaseDecoder(options) {
@@ -1209,8 +1211,18 @@ var BaseDecoder = /*#__PURE__*/function () {
       });
     }
 
-    this.bytesWritten += this.bufferWrite(buffers);
-    this.canPlay = true;
+    try {
+      this.bytesWritten += this.bufferWrite(buffers);
+      this.canPlay = true;
+    } catch (error) {
+      if (error instanceof WebAssembly.RuntimeError && error.message.indexOf('out of bounds') !== -1) {
+        if (_jsmpeg_window_reloading === false) {
+          _jsmpeg_window_reloading = true;
+          console.warn('JSMpeg: Out of bounds error. Reloading...');
+          document.location.reload();
+        }
+      }
+    }
   };
 
   _proto.seek = function seek(time) {
